frontend/src/console/components/ConsolePage/ConsolePage.tsx

```diff
@@ -30,17 +30,26 @@ interface Props extends StateProps, ActionProps {
 }
 
 export class ConsolePageComponent extends React.PureComponent<Props> {
+  refreshInterval: any | undefined;
```

Comment:
The type of this should be

```typescript
refreshInterval?: NodeJS.Timer;
```

Reply:
Done! Thanks for the tip, stronger the typing the better. :)